release-notes/9.0/supported-os.json

```diff
@@ -202,78 +170,43 @@
             ]
         },
         {
-            "family": "Windows",
-            "distributions": [
-                {
-                    "name": "Nano Server",
```

Comment:
Where do you intend to add the information about Nano Server and Windows Server Core? They have different support information, so I'm not sure if we can just refer to any of the remaining ones.